scout/models/case/case_loading_models.py

```diff
@@ -466,18 +466,19 @@ def set_synopsis(cls, synopsis: Optional[Union[str, List]]) -> Optional[str]:
             synopsis = ". ".join(synopsis)
         return synopsis
 
-    @field_validator("genome_build", mode="before")
+    @model_validator(mode="before")
     @classmethod
-    def format_build(cls, genome_build: Union[str, int]) -> str:
+    def format_build(cls, values) -> "CaseLoader":
         """Format the genome build so it will be saves as either '37' or '38'."""
-        str_build = str(genome_build)
+        str_build = str(values.get("genome_build") or values.get("human_genome_build", ""))
```

Comment:
And same here I guess, though the alias is hidden rather deep. Considering it is the key we use both in demos, docs and in CG prod I find it may need a little explanation - maybe in that docstring above?

Reply:
Fixed the docstriing